deepchem/splits/splitters.py

```diff
@@ -847,50 +847,60 @@ def split(self,
 
 class ScaffoldSplitter(Splitter):
   """
-    Class for doing data splits based on the scaffold of small molecules.
-    """
+  Class for doing data splits based on the scaffold of small molecules.
+  """
 
   def split(self,
             dataset,
-            seed=None,
```

Comment:
I removed here the argument `seed=None` because it is not used in the code. After a second thought, this might break people's code so maybe it would be a good idea to bring it back?

Reply:
I think it's good to remove it here since people might have been using this argument in the belief it was doing something for them. We should make sure to make a note about this in the release notes for DeepChem 2.4 though.